dashboard/test/ui/features/teacher_homepage.feature

```diff
@@ -63,6 +63,21 @@ Feature: Using the teacher homepage sections feature
     And I wait until element ".header_popup_body .uitest-ProgressBubble:first" is visible
     And the href of selector ".header_popup_body .uitest-ProgressBubble:first" contains the section id
 
+  Scenario: Assign a CSF course with multiple versions
+    When I see the section set up box
+    And I create a new section with course "Course A", version "2017 (Recommended)"
+    Then the section table should have 1 rows
+    And the section table row at index 0 has primary assignment path "/s/coursea"
+
+    When I click selector ".ui-test-section-dropdown"
+    And I click selector ".edit-section-details-link"
+    And I wait until element "#assignment-version-year" is visible
+    And I select the "2018" option in dropdown "assignment-version-year"
+    And I press the first ".uitest-saveButton" element
+    And I wait for the dialog to close
+    Then I should see the section table
+    And the section table row at index 0 has primary assignment path "/s/coursea-2018"
```

Comment:
FYI @madelynkasula that this adds some UI test coverage for the script update path, though only for single scripts

Reply:
this is great! the PR i'm working on now (fixing the endpoint bug and adding a feature test) will have lots of little merge conflicts with this, so i'll wait to merge until you've merged this PR